src/main/resources/META-INF/services/org.keycloak.events.EventListenerProviderFactory

```diff
@@ -0,0 +1 @@
+io.kokuwa.keycloak.metrics.MicrometerEventListenerFactory
```

Comment:
This could be autogenerated by the com.google.auto.service.AutoService; annotation at class-level